app/style/content/conversation/message-list.less

```diff
@@ -88,6 +88,18 @@
   &.accent-color-6 {color: @accent_colors-6}
   &.accent-color-7 {color: @accent_colors-7}
 }
+.message-header-label-sender {
+  font-weight: @font-weight-bold;
+  text-transform: none;
```

Comment:
Confirmed with @markesaaremets that is should not be uppercase and not capitalizing the first char either.